python/src/aac/spec/spec.yaml

```diff
@@ -616,8 +621,6 @@ schema:
       type: SpecSection[]
     - name: requirements
       type: Requirement[]
-  required:
```

Comment:
Required fields went away quite a while ago. The required fields validator made the `required` field obsolete. This appears to be a remnant that was uncovered by applying the undefined fields validator.

Reply:
Okay then, good to know. Guess they went away before my time, and these have been hanging around still. They were there when I did the validator end of last iteration because I thought I had to add an entry to the required list in the spec to get mine to work correctly from what I was seeing in other validator PRs.